app/front-end/src/components/buttons/IconTitleButton.tsx

```diff
@@ -8,6 +8,7 @@ interface Props {
   height?: string;
   borderRadius?: string;
   onClick?: () => void;
+  isActive?: boolean;
```

Comment:
Also add disabled for unusable buttons like macros to not be available

Reply:
Done!